libs/auth/src/java_card_config.ts

```diff
@@ -10,52 +11,38 @@ export interface JavaCardConfig {
   /** Only VxAdmin should provide these params, for card programming */
   cardProgrammingConfig?: {
     vxAdminCertAuthorityCertPath: string;
-    vxAdminPrivateKeyPassword: string;
-    vxAdminPrivateKeyPath: string;
+    vxAdminPrivateKey: FileKey | TpmKey;
   };
   /** Only tests should provide this param, to make challenge generation non-random */
   customChallengeGenerator?: () => string;
   /** The path to the VotingWorks cert authority cert on the machine */
   vxCertAuthorityCertPath: string;
 }
 
-/**
- * The password for all dev private keys
- */
-export const DEV_PRIVATE_KEY_PASSWORD = '1234';
```

Comment:
As noted in the last commit message, I'm removing the need for passphrases when using private key files
since those will now only be used in dev. In prod, we'll always be using TPM keys